### PR TITLE
Feature/256279 - new contacts journey action independent of event-tracks

### PR DESCRIPTION
**Pull request by mathiasfk**
- New contacts journey action independent of event-tracks.
- This will enable to expand the Contacts Journey feature to all bots wil reduced cost.

### CODE REVIEW

**Review by rodrigoxavier — APPROVED** [review]

**Review by juliacariafreitas — APPROVED** [review]

File: src/Take.Blip.Builder.UnitTests/Actions/TrackContactsJourneyTests.cs [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+            };
+
+            // Assert
+            var exception = await Should.ThrowAsync<ArgumentException>(
+                // Act
+                async () => await contactsJourneyAction.ExecuteAsync(Context, settings, CancellationToken)
+            );
+            exception.Message.ShouldContain("stateName");
+        }
+
```
Comment (EricJangola) [inline src/Take.Blip.Builder.UnitTests/Actions/TrackContactsJourneyTests.cs:141]: Unecessary enter's

File: src/Take.Blip.Builder.UnitTests/Actions/TrackContactsJourneyTests.cs [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+{
+    public class TrackContactsJourneyTests : ActionTestsBase
+    {
+        public TrackContactsJourneyTests()
+        {
+            ContactsJourneyExtension = Substitute.For<IContactsJourneyExtension>();
+        }
+
+        public IContactsJourneyExtension ContactsJourneyExtension { get; private set; }
+
```
Comment (EricJangola) [inline src/Take.Blip.Builder.UnitTests/Actions/TrackContactsJourneyTests.cs:21]: Unecessary enter

File: src/Take.Blip.Client/Extensions/ContactsJourney/ContactsJourneyExtension.cs [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+using Lime.Protocol;
+using Takenet.Iris.Messaging.Resources.Analytics;
+
+namespace Take.Blip.Client.Extensions.ContactsJourney
+{
+    public class ContactsJourneyExtension : ExtensionBase, IContactsJourneyExtension
+    {
+        private const string CONTACTS_JOURNEY_URI = "/contacts-journey";
+
+        private readonly Node AnalyticsAddress = Node.Parse("postmaster@analytics.msging.net");
```
Comment (EricJangola) [inline src/Take.Blip.Client/Extensions/ContactsJourney/ContactsJourneyExtension.cs:13]: We have a constants file with the Domain  (  see HelpDeskExtension for an example )

File: src/Take.Blip.Builder.UnitTests/Actions/TrackContactsJourneyTests.cs (hunk 1 above)
Comment (mathiasfk) [reply src/Take.Blip.Builder.UnitTests/Actions/TrackContactsJourneyTests.cs]: Done

File: src/Take.Blip.Builder.UnitTests/Actions/TrackContactsJourneyTests.cs (hunk 2 above)
Comment (mathiasfk) [reply src/Take.Blip.Builder.UnitTests/Actions/TrackContactsJourneyTests.cs:21]: Done

File: src/Take.Blip.Client/Extensions/ContactsJourney/ContactsJourneyExtension.cs (hunk 3 above)
Comment (mathiasfk) [reply src/Take.Blip.Client/Extensions/ContactsJourney/ContactsJourneyExtension.cs]: Done

**Review by EricJangola — APPROVED** [review]

**Review by jujucasseb — APPROVED** [review]

**Pull request merged**